healthScore/views.py

```diff
@@ -778,3 +778,16 @@ def request_health_history(request):
         return redirect("homepage")
 
     return JsonResponse({"error": "Unauthorized"}, status=401)
+
+
+@login_required
+@csrf_exempt
+def view_health_history_access_requests(request):
+    if request.method == "GET":
```

Comment:
Could you also please add an else block (or another return) to cover the cases where we might request using a different method using postman? And also please add a test case for that extra line

Reply:
Sure